Gemfile

```diff
@@ -1,15 +1,14 @@
 source 'https://rubygems.org'
 
-gem 'fluentd', '<=1.2.4'
-gem 'activesupport', '~>5.2.1'
-gem 'fluent-plugin-concat', '~>2.3.0'
-gem 'fluent-plugin-kubernetes_metadata_filter', '~>2.1.2'
-gem 'fluent-plugin-elasticsearch', '~>2.11.5'
-gem 'fluent-plugin-systemd', '~>1.0.1'
-gem 'fluent-plugin-detect-exceptions', '~>0.0.11'
-gem 'fluent-plugin-prometheus', '~>1.0.1'
-gem 'fluent-plugin-multi-format-parser', '~>1.0.0'
-gem 'fluent-plugin-rewrite-tag-filter', '>=2.0.0'
-gem 'fluent-plugin-record-modifier', '>=1.1.0'
-gem 'fluent-plugin-http-healthcheck', '>=0.1.0'
-gem 'oj', '~>3.6.5'
+gem 'fluentd', '~> 1.7', '>= 1.7.3'
```

Comment:
that is a big jump! :) 

did we review the changelog ?

Reply:
Of course! :)

It looked like there were no breaking changes - only additions and bugfixes. Seems to be running ok on the updated platform.